app/components/dashboard/approvals_component.html.erb

```diff
@@ -15,7 +15,7 @@
     <% approvals.each do |submission| %>
       <tr>
         <%= render Works::LinkToShowComponent.new(work_version: submission) %>
-        <td><%= submission.work.depositor.sunetid %></td>
+        <td><%= submission.work.owner.sunetid %></td>
         <td><%= render Collections::LinkToShowComponent.new(collection_version: submission.work.collection.head) %></td>
```

Comment:
not related to emails, but seems like we'd want to show the current relevant owner of the work here too